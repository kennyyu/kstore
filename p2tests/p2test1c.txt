insert(t1a,50)
insert(t1a,250)
insert(t1a,950)
X=select(t1a)
fetch(t1a,X)
insert(t2a,50)
insert(t2a,250)
insert(t2a,950)
Y=select(t2a)
fetch(t2a,Y)
insert(t2c,10)
Z=select(t2c,8,10)
fetch(t2c,Z)
