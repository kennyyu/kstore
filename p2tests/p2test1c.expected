50
100
200
250
300
400
500
600
700
800
900
950
50
100
200
250
300
400
500
600
700
800
900
950
8
9
10
